setup.cfg

```diff
@@ -16,6 +17,10 @@ install_requires =
   attrs
 
 
+[options.package_data]
+matrix_common = py.typed
```

Comment:
Not seen this before.
My approach was to use `MANIFEST.in`.

http://blog.codekills.net/2011/07/15/lies,-more-lies-and-python-packaging-documentation-on--package_data-/ suggests that `package_data` is full of lies and you should prefer `MANIFEST.in` since that works for both sdist and bdists whereas `package_data` doesn't work for source distributions.

The modern way to do both a sdist and a bdist seems to be `python -m build --sdist --wheel`. With the lines you have written, both options seem to include this file. As a result, I think the concern above may be historical.
(N.B. seems like you need to reinstall the package in the venv before changes to `setup.cfg` take effect)

Reply:
I'm a bit confused about the difference between `MANIFEST.in` and `package_data` now. The latter seems to do the right thing for us but there are a bunch of posts suggesting to use the former.

I'll go with `package_data` for now since it seems to work.